sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationAsyncClientBuilder.java

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.attestation;
+
+import com.azure.core.annotation.ServiceClientBuilder;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.http.policy.RetryPolicy;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.core.util.serializer.SerializerAdapter;
+import com.azure.security.attestation.implementation.AttestationClientImpl;
+import com.azure.security.attestation.implementation.AttestationClientImplBuilder;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Objects;
+
+/** A builder for creating a new instance of the AttestationClient type. */
+@ServiceClientBuilder(
+        serviceClients = {
+            AttestationAsyncClient.class,
+            SigningCertificatesAsyncClient.class
+        })
+public final class AttestationAsyncClientBuilder {
```

Comment:
We only have one builder for both sync\async clients and expose buildAsyncClient and buildSyncClient methods on it.

Sample - [https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRegistryClientBuilder.java](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRegistryClientBuilder.java)

Reply:
Sorry, that was my confusion.

Based on our discussions I thought that the standard required an async builder and a sync builder, one of which creates async clients the other which creates sync clients. I'll remove the async builder and fold it into the builder.

Reply:
Looking at the .Net ApiView - I only see a AttestationClient and no SigningCertificatesClient - can you tell me how you are exposing that functionality in .Net?

Reply:
Also, in that ApiView I see an AttestationAdministrationClient which I do not see here.